app/helpers/dashboards_helper.rb

```diff
@@ -16,9 +16,10 @@ def voicemail_options
     enum_text = { not_specified: 'No instructions; no ID VM',
                   no: 'Do not leave a voicemail',
                   yes: 'Voicemail OK, ID OK' }
-    vm_options = Patient::VOICEMAIL_PREFERENCE
+    vm_options = Patient.voicemail_preference.values
```

Comment:
From what I remember, the view depended on having symbols on the erb.html code to render 'MD','VA', etc. I tried using strings and could never get them showing up.